.github/CONTRIBUTING.md

```diff
@@ -71,6 +71,38 @@ This project and everyone participating in it is governed by the
   smaller PRs that can be reviewed and merged independently. This makes it easier for
   reviewers to understand the changes and for maintainers to merge the PR.
 
+There are several code analysis tools, that are executing regularly: 
```

Comment:
Please remove it, there a way more tools behind.
No need to mention all of them